playwright/e2e/crypto/utils.ts

```diff
@@ -7,8 +7,9 @@ Please see LICENSE files in the repository root for full details.
 */
 
 import { expect, JSHandle, type Page } from "@playwright/test";
+import { ICreateRoomOpts, MatrixClient } from "matrix-js-sdk/src/matrix";
+import { CryptoEvent } from "matrix-js-sdk/src/crypto-api";
```

Comment:
Switching from `import type` to `import` isn't a great idea here.

Reply:
Oh good catch